Core/source/core/pgp-key.ts

```diff
@@ -196,13 +205,14 @@ export class PgpKey {
       }
       for (const k of keys) {
         for (const u of k.users) {
+          await PgpKey.validateAllDecryptedPackets(k);
           u.otherCertifications = []; // prevent key bloat
         }
       }
-      return { normalized: keys.map(k => k.armor()).join('\n'), keys };
+      return { normalized: keys.map(k => k.armor()).join('\n'), keys, exception: '' };
```

Comment:
no need to return `exception` here if it is declared optional

Reply:
updated, thanks!